tests/lti/views/export_test.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+import urllib
+import urlparse
+
+from pyramid.view import view_config
+
+from pyramid.httpexceptions import HTTPFound
+
+from lti import util
+
+import lti.views.export
+
+def test_it_returns_HTTPFOUND(pyramid_request):
```

Comment:
I'd rename this to something like `test_it_redirects_to_the_export_facet_static_file()`. Something that says _why_ it's returning an `HTTPFound`. If we later changed the code to do the redirect in some other way, instead of returning an `HTTPFound`, we would have to update this test but at least the test name could remain the same.